libvast/include/vast/plugin.hpp

```diff
@@ -402,6 +403,47 @@ class rest_endpoint_plugin : public virtual plugin {
     = 0;
 };
 
+// -- loader plugin -----------------------------------------------------
+
+using input_loader = std::function<auto()->generator<chunk_ptr>>;
+using input_parser = std::function<auto(chunk_ptr)->generator<table_slice>>;
+using options = record;
```

Comment:
This was marked "exposition only" in the design doc as we weren't sure on the shape of the options yet; please just use `const record&` directly below and remove this alias.

Reply:
I removed the alias in https://github.com/tenzir/vast/pull/2935/commits/405db6875bd7a26ce4bb440f9010f8335dc84413